src/gui/torrentcreatordialog.cpp

```diff
@@ -185,9 +186,9 @@ void TorrentCreatorDialog::onCreateButtonClicked()
 {
 #ifdef Q_OS_WIN
     // Resolve the path in case it contains a shortcut (otherwise, the following usages will consider it invalid)
-    const auto inputPath = Utils::Fs::toCanonicalPath(Path(m_ui->textInputPath->text().trimmed()));
```

Comment:
The `trimmed` is removed since filename with trailing whitespaces is valid in linux. And thanks to `FileSystemPathLineEdit` we can directly see if the path is valid or not and not worry about the spaces now.